### DISCUSSION TITLE
下载渲染工具是出现问题怎么办？

### DISCUSSION THREAD

**Discussion by mbwsfy0110** [YunYouJun/hexo-theme-starry#356] [Q&A 疑难杂症]
你好，我下载渲染工具的时候，下边会显示“Some issues need review, and may require choosing
a different dependency.”，请问这个是为什么呢？
$ npm install hexo-render-pug hexo-renderer-stylus --save
npm WARN deprecated core-js@2.6.12: core-js@❤️.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.

added 61 packages, and audited 305 packages in 16s

24 packages are looking for funding
run npm fund for details

2 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.

Run npm audit for details.

**Comment by mbwsfy0110** [comment]
知道了，打扰了，哈哈哈

**Reply by YunYouJun** [reply]
渲染器依赖版本过旧，不影响使用。

**Reply by mbwsfy0110** [reply]
> 渲染器依赖版本过旧，不影响使用。

学到了，感谢！🤭
